game.rb

```diff
@@ -0,0 +1,12 @@
+class Game
```

Comment:
From the diagram, the `Game` class is a child of the `Item` class. Please make the `Game` class inherit from the `Item` class. `class Game < Item`